workflow-ui/core-android/src/test/java/com/squareup/workflow1/ui/CompositeViewRegistryTest.kt

```diff
@@ -6,9 +6,9 @@ import kotlin.reflect.KClass
 import kotlin.test.assertFailsWith
 
 @OptIn(WorkflowUiExperimentalApi::class)
-class CompositeViewRegistryTest {
+public class CompositeViewRegistryTest {
```

Comment:
We don't need public visibility modifiers in tests. They are explicitly excluded from the gradle plugin, Android Studio is just clueless.

Reply:
Yeah, but it's also drowning out all the real warnings. Not sure what to do instead.

Reply:
Out of scope for this PR so I'll back them out.